src/main/java/it/gov/pagopa/payhub/activities/util/Utilities.java

```diff
@@ -125,6 +128,14 @@ public static OffsetDateTime toOffsetDateTimeEndOfTheDay(LocalDate localDate) {
         return endOfDay.atZone(ZONEID).toOffsetDateTime();
     }
 
+    public static OffsetDateTimeIntervalFilter toOffsetDateTimeIntervalFilterForDayBounds(LocalDateIntervalFilter localDateIntervalFilter){
```

Comment:
```suggestion
    /** It will convert a {@link LocalDateIntervalFilter} into an {@link OffsetDateTimeIntervalFilter} configured from the start of the <i>from</i> day to the end of the <i>to</i> day */
    public static OffsetDateTimeIntervalFilter toRangeClosedOffsetDateTimeIntervalFilter(LocalDateIntervalFilter localDateIntervalFilter){
```